packages/engine/errors/2571.md

```diff
@@ -0,0 +1,6 @@
+---
+original: "Object is of type 'unknown'."
+excerpt: "This variable is unknown so I don't know what type it or if is in it."
```

Comment:
"unknown" === "I don't know what it is" 😅 
The excerpt is a bit redundant imho.
```suggestion
excerpt: "I cannot access properties of a variable whose type I don't know"
```

The description below could be helpful at explaining the user what to do:
````
---

You need to narrow down the type of the variable before accessing its properties or assigning it to a different typed variable. For example:

```ts
const foo: unknown = "bar";

if (typeof foo === "string") { // Now I know 'foo' is of type 'string',
  const baz: string = foo;     // I can assign it to a typed variable
  foo.toUpperCase();           // and access its properties
}
```

````

Reply:
way better!